src/pyJiraCli/cmd_profile.py

```diff
@@ -215,12 +212,7 @@ def _update_profile(args) -> Ret.CODE:
         if args.cert is not None:
             ret_status = _profile.add_certificate(args.profile_name, args.cert)
 
-        if args.token is not None:
-            ret_status = _server.try_login(_profile.get_server_url(),
```

Comment:
the server is needed here to check the validity of a token or certificate

Reply:
Certificate will not be checked for validity by adding it to the profile. It will be saved and the validity will be checked the next time the tool is called